templates/src/csharp/Grpc.Core.Tests/project.json.template

```diff
@@ -12,12 +12,13 @@
     },
     "frameworks": {
       "net45": { },
-      "netstandard1.5": {
+      "netcoreapp1.0": {
         "imports": [
           "portable-net45"
         ],
         "dependencies": {
-          "NETStandard.Library": "1.5.0-rc2-24027"
+          "Microsoft.NETCore.App": "1.0.0",
+          "NETStandard.Library": "1.6.0"
```

Comment:
just a last couple things, since Microsoft.NETCore.App brings it in, I think we can drop the NETStandard.Library dependency?